src/components/Restaurant/RestaurantHeader.tsx

```
@@ -245,32 +248,28 @@ const RestaurantHeader: React.FC<RestaurantHeaderProps> = ({
           className="fixed inset-0 bg-black/50 flex items-center justify-center p-4"
           onClick={() => setShowUnfollowModal(false)}
         >
           <div className="bg-white rounded-xl w-full max-w-sm overflow-hidden p-8">
             <h3 className="text-lg font-semibold text-gray-900 mb-2">
-              Unfollow Restaurant
+              {t('unfollowmodal.title')}
             </h3>
-            <p className="text-gray-600 mb-6">
-              Are you sure you want to unfollow <strong>{name}?</strong>
-            </p>
+            <p className="text-gray-600 mb-6" dangerouslySetInnerHTML={{ __html: t('unfollowmodal.prompt', { name }) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)